indexer/indexer.go

```diff
@@ -0,0 +1,26 @@
+package indexer
```

Comment:
Might be less change and easier to merge upstream changes in the future if this was called something different...

Reply:
good point. I spent a bit of time thinking about the name and ended up with that because it aligns with other components having their own packages (find, ingest, registry, admin, etc.).

I'll think a bit more about it, suggestions are welcome.